oat-sa/extension-tao-testqti
# AUT-1831 fix: action icons selection and behaviour

Related to: https://oat-sa.atlassian.net/browse/AUT-1831

**Steps to reproduce:**

- Go to Tests tab
- Create a new test and go to it's Authoring
- Add subsections up to 3rd level
- Click 'Move Down' (arrow down icon) on the 2nd level section

**Actual result:** All nested section are hidden.
**Expected result:** Nothing happened since there's no other sections and the icon is disabled.
